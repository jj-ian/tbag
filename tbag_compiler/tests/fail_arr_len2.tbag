true {
	int a;
        int b;
	int len;
	len = arrLen(a, b);
	print(len);
	endgame;
}
