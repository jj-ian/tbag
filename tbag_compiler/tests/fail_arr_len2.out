Fatal error: exception Failure("Function arrLen does
                            not exist with the given parameters.")
